pypesto/sampling/pymc3.py

```diff
@@ -56,44 +61,32 @@ def initialize(self, problem: Problem, x0: np.ndarray):
 
         self.problem.objective.history = History()
 
-    def sample(
-            self, n_samples: int, beta: float = 1.
-    ):
-        problem = self.problem
-        log_post_fun = TheanoLogProbability(problem, beta)
-        trace = self.trace
+    def sample(self, n_samples: int, beta: float = 1.):
+        # step, by default automatically determined by pymc3
+        step = None
+        if self.step_function:
+            step = self.step_function()
 
-        x0 = None
-        if self.x0 is not None:
-            x0 = {x_name: val
-                  for x_name, val in zip(self.problem.x_names, self.x0)}
+        # Create PyMC3 model
+        model = PyMC3Model(self.problem, self.x0, beta,
```

Comment:
where is this class defined?

Reply:
Leftover from older commit, see discussion about `create_pymc3_model`.